python/paddle/fluid/dygraph/jit.py

```diff
@@ -54,14 +56,12 @@ def _dygraph_to_static_output_(dygraph_func):
     def __impl__(*args, **kwargs):
         # Get AST from dygraph function
         dygraph_code = inspect.getsource(dygraph_func)
+        dygraph_code = textwrap.dedent(dygraph_code)
         root = gast.parse(dygraph_code)
 
-        root = DygraphToStaticAst().get_static_ast(root)
+        root, func_name = DygraphToStaticAst().get_static_ast(root)
+        static_func, file_name = ast_to_func(root, func_name)
```

Comment:
Is it better that the return value of `get_static_ast` only include ast.node because "get_static_ast" means get static ast ?

Reply:
Thanks, fixed